src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -726,18 +750,7 @@ def mod(param_set):
     )
     def test_run_app_bulk__good_inputs(self, auth, c):
         c.return_value.send_comm_message = MagicMock()
-        test_input = self.bulk_run_good_inputs
-
-        child_job_params = [
-            copy.deepcopy(test_input[0]["params"][0]),
-            copy.deepcopy(test_input[0]["params"][1]),
-            copy.deepcopy(test_input[1]["params"][0]),
-        ]
-        for param_set in child_job_params:
-            for key, value in param_set.items():
-                if value == "":
-                    param_set[key] = None
-            param_set["workspace_name"] = self.public_ws
```

Comment:
This is why I thought I had to make a global `WS_NAME` variable, because I was refactoring out spec-mapped parameter sets, and here it was using the config workspace name. But all this spec-mapping code turned out to be unnecessary